### PR TITLE
arch/arm64: cleanup defconfigs

### PR DESCRIPTION
**Pull request by HidenoriMatsubayashi**
## Summary
Use `CONFIG_ARCH_CHIP_QEMU` instead of `CONFIG_ARCH_CHIP_QEMU_A53`. This is because these configurations depend on the hardware configuration of qemu (memory map, irq numbers, etc.). They independent of CPU core types.

Memory map for qemu virtio:
https://github.com/qemu/qemu/blob/master/hw/arm/virt.c

## Impact
None

## Testing
Confirmed that OS booted with all armv8a cpus that qemu supports.

cortex-a53:
```shell
 qemu-system-aarch64 -cpu cortex-a53 -smp 4 -nographic -machine virt,virtualization=on,gic-version=3 \
   -net none -chardev stdio,id=con,mux=on -serial chardev:con -mon chardev=con,mode=readline -kernel ./nuttx
```

cortex-a57:
```shell
 qemu-system-aarch64 -cpu cortex-a57 -smp 4 -nographic -machine virt,virtualization=on,gic-version=3 \
   -net none -chardev stdio,id=con,mux=on -serial chardev:con -mon chardev=con,mode=readline -kernel ./nuttx
```

cortex-a72:
```shell
 qemu-system-aarch64 -cpu cortex-a72 -smp 4 -nographic -machine virt,virtualization=on,gic-version=3 \
   -net none -chardev stdio,id=con,mux=on -serial chardev:con -mon chardev=con,mode=readline -kernel ./nuttx
```

Signed-off-by: Hidenori Matsubayashi <hidenori.matsubayashi@gmail.com>

### CODE REVIEW

**Comment by HidenoriMatsubayashi** [comment]
Hi @MasayukiIshikawa @xiaoxiang781216

Can you please review?

**Comment by HidenoriMatsubayashi** [comment]
Or, perhaps we should create a new config for qemu virt (e.g. CONFIG_ARCH_CHIP_QEMU_VIRT).

**Review by xiaoxiang781216 — APPROVED** [review]

**Comment by xiaoxiang781216** [comment]
> Or, perhaps we should create a new config for qemu virt? (e.g. `CONFIG_ARCH_CHIP_QEMU_VIRT`).

it's more simple to remove _A53 suffix.

**Comment by xiaoxiang781216** [comment]
let's ignore the temp ci broken and merge this simple change directly.

**Comment by xiaoxiang781216** [comment]
> > Or, perhaps we should create a new config for qemu virt? (e.g. `CONFIG_ARCH_CHIP_QEMU_VIRT`).
> 
> it's more simple to remove _A53 suffix.

@HidenoriMatsubayashi could you remove all _A53, if the code work with other armv8a core.

**Comment by HidenoriMatsubayashi** [comment]
> could you remove all _A53, if the code work with other armv8a core.

Ok, I'll do try that.

**Comment by masayuki2009** [comment]
@HidenoriMatsubayashi 
Please use @masayuki2009 the next time.

**Comment by masayuki2009** [comment]
>let's ignore the temp ci broken and merge this simple change directly.

@xiaoxiang781216 
Do you know why the CI breaks now?

File: boards/arm64/qemu/qemu-a53/configs/nsh/defconfig [hunk 1]
```diff
@@ -11,7 +11,6 @@ CONFIG_ARCH_BOARD="qemu-a53"
 CONFIG_ARCH_BOARD_QEMU_A53=y
 CONFIG_ARCH_CHIP="qemu"
 CONFIG_ARCH_CHIP_QEMU=y
-CONFIG_ARCH_CHIP_QEMU_A53=y
```
Comment (xiaoxiang781216) [inline boards/arm64/qemu/qemu-a53/configs/nsh/defconfig:14]: @HidenoriMatsubayashi need change Kconfig too, I think.

Comment (HidenoriMatsubayashi) [reply boards/arm64/qemu/qemu-a53/configs/nsh/defconfig:14]: Thanks. Fixed.

Comment (HidenoriMatsubayashi) [reply boards/arm64/qemu/qemu-a53/configs/nsh/defconfig:14]: By the way, I think we should also change the name `CONFIG_ARCH_BOARD_QEMU_A53` (+ directory name). Shall I respond with another PR?

Comment (xiaoxiang781216) [reply boards/arm64/qemu/qemu-a53/configs/nsh/defconfig:14]: I think we can fix in one patch.

Comment (HidenoriMatsubayashi) [reply boards/arm64/qemu/qemu-a53/configs/nsh/defconfig:14]: OK, I'll fix it.

**Review by jerpelea — APPROVED** [review]

**Review by xiaoxiang781216 — APPROVED** [review]

**Comment by xiaoxiang781216** [comment]
Let's ignore the ci temp break.

**Comment by HidenoriMatsubayashi** [comment]
Thanks for the reviews.

> I think we should also change the name CONFIG_ARCH_BOARD_QEMU_A53 (+ directory name).

I'll create a new patch and send it later.

**Pull request merged**